pkg/adapter/adapter.go

```diff
@@ -112,7 +111,7 @@ func (a *Adapter) RequestPageFromDatasource(
 			}...,
 		),
 
-		// SCAFFOLDING:
+		// SCAFFOLDING #25 - pkg/adapter/adapter.go: Uncomment to enable default datetime parsing format
```

Comment:
This snippet is more related to setting the default timezone in case timezone info is absent in the ingested datetime attribute than specifying the datetime format. The format is specified in SCAFFOLDING #24

Reply:
@r-rakshith done; updated the comment.